src/main/java/com/ibm/etcd/client/utils/RangeCache.java

```diff
@@ -180,9 +181,10 @@ public RangeCache(EtcdClient client, ByteString fromKey, ByteString toKey, boole
                     .get(newModsReq).get(otherKeysReq)
                     .backoffRetry(() -> !closed)
                     .timeout(300_000L).async(); // long timeout (5min) for large ranges
-            rrfut = Futures.transform(trf, (Function<TxnResponse,List<RangeResponse>>) tr
-                    -> tr.getResponsesList().stream().map(r
-                            -> r.getResponseRange()).collect(Collectors.toList()));
+            rrfut = Futures.transform(
```

Comment:
maybe collapse _some_ of these lines? :)

Reply:
Done